general/administration/hardware-acceleration.md

```diff
@@ -480,6 +480,46 @@ AMD does not provide official `amdgpu-pro` driver support for Arch Linux, but fo
 > [!NOTE]
 > For RPi3 in testing, transcoding was not working fast enough to run in real time because the video was being resized.
 
+### Configuring VAAPI on Raspberry Pi 3 and 4 running Ubuntu server
+
+1. Enable the `render` device on your Pi:
```

Comment:
I've been trying to avoid colons in favor of normal punctuation with periods and such.

Reply:
I have removed some colons. What do you think now?